XRTK-Core/Packages/com.xrtk.core/Runtime/Services/MixedRealityToolkit.cs

```diff
@@ -162,15 +162,15 @@ public void ResetProfile(MixedRealityToolkitRootProfile profile)
         public static IReadOnlyDictionary<Type, IMixedRealitySystem> ActiveSystems => activeSystems;
 
         // ReSharper disable once InconsistentNaming
-        private static readonly List<Tuple<Type, IMixedRealityService>> registeredMixedRealityServices = new List<Tuple<Type, IMixedRealityService>>();
+        private static readonly List<Tuple<Type, IMixedRealityService>> activeSystemsDataProviders = new List<Tuple<Type, IMixedRealityService>>();
```

Comment:
I think we should keep the name the same

Reply:
But they are not RegisteredServices and will never contain such?

Reply:
I like both names really. This is an internal private field anyways, so it's not as important as an public API. Even if it was, both names work for me. So I am not sure why I'm even making a comment :)

Reply:
Data providers is too specific. Should continue to be services

Reply:
Ok, fair enough, I'll rename it back

Reply:
Reverted

Reply:
Thanks. Remember that System implies a single instance, where as a service implies there could be many.